src/v2/cookbook/form-validation.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Form Validation
+type: cookbook
+order: 1.2
+---
+
+## Simple Example
+
+Form validation is natively supported by the browser, but sometimes different browsers will handle things in a manner which makes relying on it a bit tricky. Even when validation is supported perfectly, there may be times when custom validations are needed and a more manual, Vue-based solution may be more appropriate. Let's begin with a simple example.
+
+For my first example, let's create something as simple as possible. Given a form of three fields, make two required. Let's look at the HTML first:
+
+``` html
+<form id="app" @submit="checkForm" action="/something" method="post">
+  
+  <p v-if="errors.length">
+    <b>Please correct the following error(s):</b>
+    <ul>
+      <li v-for="error in errors">{{ error }}</li>
+    </ul>
+  </p>
+  
+  <p>
+  <label for="name">Name<label>
+  <input type="text" name="name" id="name" v-model="name">
+  </p>
+
+  <p>
+  <label for="age">Age<label>
+  <input type="number" name="age" id="age" v-model="age">
+  </p>
+
+  <p>
+  <label for="movie">Favorite Movie<label>
+  <select name="movie" id="movie" v-model="movie">
+    <option>Star Wars</option>
+    <option>Vanilla Sky</option>
+    <option>Atomic Blonde</option>
+  </select>
+  </p>
+
+  <p>
+  <input type="submit" value="Submit">  
+  </p>
+
+</form>
+```
+
+Let's cover it from the top. The form tag has an ID that we'll be using for the Vue component. There's a submit handler that you'll see in a bit, and the action is a fake URL that would point to something real on a server someplace (where you have backup server-side validation of course).
+
+Beneath that there isa paragraph that shows or hides itself based on an error state. This is a personal preference of mine when building forms. I like a nice simple list of errors on top of the form. You may like error handling by the fields themselves. Use what works. Also note I'm going to fire my validation on submit rather than as every field is modified. Again, this is a personal preference.

```

Comment:
Separate "isa" to "is a". Please remove the references to personal preferences, this is a cookbook, so this is implied. Please switch the "I" statements to "we".